.github/workflows/unit-tests-refresh.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   # we do both builds and all tests in a single run, so that we only upload the containers on success
   refresh:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Note that this is not actually necessary to fix the container refresh. But as it works, it can't hurt anyway to jump to 22.04 now -- it'll be around for a while.